src/bot/libraries/documentation.ts

```diff
@@ -61,21 +61,25 @@ export class Documentation {
      * Returns a string with the full detailed help information for a command.
      */
     public static getCommandHelp(command: Command) : RichEmbed {
+        let fields = [{
+            name: 'Usage',
+            value: '`' + this.getInlineUsage(command) + '`'
+        }];
+
+        // If there are arguments, add it to the fields
+        if (command.getArguments().length > 0) {
```

Comment:
```suggestion
        if (command.getArguments().length) {
```
You don't need to check explicitly if the length is greater than zero, as zero will evaluate to false and anything other than zero will evaluate to true.